src/components/ErrorFallback.tsx

```diff
@@ -0,0 +1,28 @@
+import { Component, type ReactNode } from 'react';
+
+interface Props {
+  error: Error | null;
+  onReset: () => void;
+}
+
+class ErrorFallback extends Component<Props> {
+  render(): ReactNode {
+    const { error, onReset } = this.props;
+    return (
+      <div className="w-full p-10 text-center">
+        <h2>Something went wrong</h2>
+        <p className="py-5 font-bold">
+          {error?.message || 'Unknown error occured'}
```

Comment:
```suggestion
          {error?.message || 'Unknown error occurred'}
```
use code spell checker extension